app/build.gradle.kts

```diff
@@ -1,6 +1,6 @@
 plugins {
     id("application")
-    alias(libs.plugins.th2.component)
+    id("com.exactpro.th2.gradle.component")
```

Comment:
note: Why have you decided to go back with declaring id in the gradle file instead of version catalog?

Reply:
Reverted. I have unintended incorrectly your comment 
> note: When we add plugins here with apply false this is done to specify the version only in one place. If we use dependency catalog there is no point in doing that - we are using the same version in all places automatically